apps/web/pages/booking/[uid].tsx

```diff
@@ -268,7 +268,11 @@ export default function Success(props: SuccessProps) {
       }
       return t("needs_to_be_confirmed_or_rejected" + titleSuffix);
     }
-    return t("emailed_you_and_attendees" + titleSuffix);
+    if (bookingInfo.attendees.length > 1) {
+      return t("emailed_you_and_attendees" + titleSuffix);
+    } else {
+      return t("email_all_details" + titleSuffix);
```

Comment:
I don't see `email_all_details_recurring` to be available in common.json.
Also better key name might be `email_you`.

Reply:
For recurring bookings also we need dynamic message? same as single booking?

Reply:
Yeah, the fix is supposed to handle all cases where we mention "other attendees" when infact there are no other attendees. It can be 
- For Recurring Booking
- For One time Booking
- In this file
- Any other file
- Other keys that I have not mentioned.

Reply:
@hariombalhara I guess I have made the required changes. I also checked that in my local. Can you please go through the changes?